contracts/VestingEscrowFactory.vy

```diff
@@ -1,79 +1,209 @@
-# @version 0.2.16
+# @version 0.3.7
+
 """
 @title Vesting Escrow Factory
-@author Curve Finance, Yearn Finance
+@author Curve Finance, Yearn Finance, Lido Finance
 @license MIT
```

Comment:
Can we change to GPLv3? or MIT license couldn't be changed to GPLv3?

Reply:
We can

Reply:
Changed to GPL-3.0